onelogin_rules/onelogin_high_risk_failed_login.py

```diff
@@ -0,0 +1,16 @@
+def rule(event):
+
+    # check risk associated with this event
+    if event.get('risk_score', 0) > 50:
+        # a failed authentication attempt with high risk
+        return event.get('event_type_id') == 6
+    return False
+
+
+def dedup(event):
```

Comment:
I would drop this `dedup` function. I won't repeat this comment a bunch of times, just generally anywhere that has a dedup and title function relying on the same fields in the event we can drop the dedup function.